### ISSUE TITLE
cookieDomain setting not possible

### ISSUE THREAD

**Issue by thomaszbz** [mblaschke/TYPO3-metaseo#124]
by Wolfgang Maschke via https://forge.typo3.org/issues/63505 , reported against 6.2.

Using Universal Analytics I can set plugin.metaseo.services.googleAnalytics.domainName to auto or single domain but in source code it will end like this:

```
ga('create', 'UA-1111111-1', {
'cookieDomain': '{gaDomainName}',
'cookieName': '_gua',
'cookieExpires': 20000
});
```

Any idea what to do?

Thx for your help.

Wolfgang

**Comment by thomaszbz** [comment]
[Example](https://developers.google.com/analytics/devguides/collection/analyticsjs/) suggests to use

    ga('create', 'UA-XXXXX-Y', 'auto');

**Comment by thomaszbz** [comment]
Confirmed for Metaseo 2.0.1, TYPO3 7.6.14:

                ga('create', 'UA-12345456-1', {
                    'cookieDomain': '{gaDomainName}',
                    'cookieName': '_gua',
                    'cookieExpires': 20000
                });

which is `{gaDomainName}` instead of `example.com`

**Comment by thomaszbz** [comment]
Thanks for catching this. Will be fixed in 2.0.2.
